docs/tutorials/_index.md

```diff
@@ -3,4 +3,62 @@ title: "Tutorials"
 linkTitle: "Tutorials"
 weight: 65
 type: docs
+no_list: true
 ---
+
+{{< alert title="Tip" color="tip" >}}
+While tutorials may list specific hardware, the hardware only affects the configuration of a project.
+Once you have configured your components, you can use the same code of the tutorials with any hardare.
+
+For this reason, we list tutorials based on features you may be looking for.
+{{< /alert >}}
+
+## Your first robots
+
+{{< cards >}}
+    {{< card link="/tutorials/get-started/try-viam-sdk" size="small">}}
+    {{< card link="/tutorials/get-started/make-an-led-blink-with-the-viam-app/" size="small" >}}
+    {{< card link="/tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk/" size="small" >}}
+{{< /cards >}}
+
+## Configuration and Testing
+
+{{< cards >}}
+    {{< card link="/tutorials/configure/build-a-mock-robot/" size="small" >}}
+    {{< card link="/tutorials/configure/foam-dart-launcher/" size="small">}}
+    {{< card link="/tutorials/configure/scuttlebot/" size="small" >}}
+{{< /cards >}}
+
+## Controlling
+
+{{< cards >}}
+    {{< card link="/tutorials/control/scuttle-gamepad/" size="small" >}}
+    {{< card link="/tutorials/control/yahboom-rover/" size="small" >}}
+{{< /cards >}}
+
+## Services
```

Comment:
This is okay but maybe we should keep in mind that motion is about to have a bunch of ones? and subdivide into motion and vision? or not idk

Reply:
Agree, but I think we can divide later

Reply:
agreed will keep as is for now